atomicapp/nulecule/main.py

```diff
@@ -64,6 +63,7 @@ def __init__(self, app_spec, destination=None,
             destination: where to unpack a nulecule to if it isn't local
             cli_answers: some answer file values provided from cli args
             answers_file: the location of the answers file
+            cli (dict): CLI data
         """
         self.answers = copy.deepcopy(DEFAULT_ANSWERS)
         self.cli_answers = cli_answers
```

Comment:
I was just not touching this vars to keep the changes minimal, but we have already done a lot of changes, so, I will go ahead and clean up this bit as well.